router.go

```diff
@@ -284,10 +289,14 @@ func (r *Router) pathChanged(path string, initial bool) {
 //   Route 2: /todos/{category}
 // And the path argument is "/todos/work", the bestRoute would be todos/work
 // because the string "work" matches the literal in Route 1.
-func (r Router) findBestRoute(path string) (bestRoute *route, tokens []string) {
+func (r Router) findBestRoute(path string) (bestRoute *route, tokens []string, params map[string][]string) {
+	parts := strings.Split(path, "?")
+	if len(parts) > 2 {
+		log.Println("Path has more than two parts: " + path)
```

Comment:
Was this supposed to be left here? I'm sure this scenario warrants a log message.